examples/faraday_redirect_options.rb

```diff
@@ -0,0 +1,32 @@
+# A basic MetaInspector example for scraping a page
+#
+# Usage example:
+#
+#   ruby faraday_redirect_options.rb http://facebook.com
+
+require '../lib/metainspector'
```

Comment:
We need to add `require 'resolv'` to run this example.

Reply:
Ups! Missed that 😅